plugins/Google/plugin.py

```diff
@@ -446,6 +446,17 @@ def phonebook(self, irc, msg, args, phonenumber):
             irc.reply(_('Google\'s phonebook didn\'t come up with anything.'))
     phonebook = wrap(phonebook, ['text'])
 
+    _gautocompleteUrl = 'https://suggestqueries.google.com/complete/search?client=firefox&q=%s'
+    @wrap(['text'])
+    def autocomplete(self, irc, msg, args, text):
+        """ <text> """
+        aut = json.loads(
+            utils.web.getUrl(self._gautocompleteUrl % utils.web.urlquote_plus(text)))[1]
```

Comment:
I think that's better to ensure that json.loads() is non returning an empty object; should it happen that indexing will fail